core/modules/node/node.pages.inc

```diff
@@ -382,16 +385,21 @@ function node_form($form, &$form_state, Node $node) {
     $form['options']['status'][NODE_SCHEDULED]['#access'] = FALSE;
   }
 
+  $form['options']['additional'] = array(
+    '#type' => 'markup',
+    '#markup' => $node->nid ? t('<label>Additional options</label>') : t('<label>Additional actions</label>'),
```

Comment:
We should be avoiding adding HTML tags in translatable strings. The `<label>` tag can be added to the element using `#prefix`/`#suffix`, like so:

```suggestion
    '#prefix' => '<label>',
    '#markup' => $node->nid ? t('Additional options') : t('Additional actions'),
    '#suffix' => '</label>',
```